dashboard/test/lib/contact_rollups_v2_test.rb

```diff
@@ -5,12 +5,14 @@
 class ContactRollupsV2Test < ActiveSupport::TestCase
   setup do
     EmailPreference.delete_all
```

Comment:
@wjordan We're interested in your view here. Will these specific cleanups interfere with other tests? AFAIK, tests targeting the same database are running sequentially, so there shouldn't be any interferences?

Reply:
I think that's correct- with the current parallel test setup each process has its own unique database and tests run on a single thread in each process, so there shouldn't be any interference from tests concurrently querying the same database.

Some other details worth noting about how Dashboard (Rails) tests work:
- By default Rails wraps each individual test in a transaction that's rolled back at the end, so a separate `teardown` 
block to cleanup leftover data after each test may not be necessary.
- For the same reason, in normal operation there shouldn't be any 'stale' data in the database needing cleanup before the test begins, the only data present would be 'fixtures' loaded at test initialization, so the `setup` block to cleanup existing data may not even be necessary either.
- We run two Rails patches to enable an extra config `self.use_transactional_test_case = true` that wraps _all_ tests in the test-case class in a transaction that's rolled back (in addition to each individual test still being run in a sub-transaction that's rolled back), so you can run a `setup_all` block to initialize some database state that will be used by all tests in the class.
- Beyond that, in general I would recommend designing tests that can still run even if the database has some unrelated data if possible, so that it doesn't depend on specific autoincrement ids of objects that are created/deleted. (For example, asserting some property matches the ID of a newly-created object, rather than asserting some property is 1). The tests will be more robust (and future-proof, for example if we ever wanted to try running multiple test-threads per process).

Reply:
Thanks, Will! It all makes sense. I'm okay with removing both `setup` and `teardown`, which just do database cleanup.